documentation/getting_started/configuring-address-spaces.adoc

```diff
@@ -0,0 +1,38 @@
+[id='getting-started-configuring-address-spaces-{context}']
+= Configuring Address Spaces
+
+{ProductName} support the `brokered` and `standard` address spaces. This guide will use the
+`standard` address space. See <<address_model>> for more information about the differences.
```

Comment:
For more information about the differences, see xref:con-address-space-messaging[].